scripts/travis_script.sh

```diff
@@ -30,4 +30,17 @@ if [ ${TASK} == "cmake-build" ]; then
     cd build
     cmake ..
     make all || exit -1
-fi
\ No newline at end of file
+fi
+
+if [ ${TASK} == "xgb-java-tests" ]; then
```

Comment:
are we really going to do this?

it sounds strange to me....Spark relies on netty but netty never put Spark as a integration test, 

@hcho3  what do you think

Reply:
@CodingCat This is something I've been thinking about, about the need for integration tests for dmlc-core / rabit against upstream projects. The question is where is the correct place to perform such tests. Should Rabit host integration tests for XGBoost, or should XGBoost do it? Or should we create a separate GitHub repo to do inter-project integration tests?

Reply:
removed xgb java tests from rabit. So integration test fall back to where we started and only runs when we try to upgrade rabit in xgboost. I incline to keep this process as simple as possible with cost of manual work on upgrade rabit from time to time.

Reply:
I think XGBoost should do an integration test covering all involved rabit functionalities

Rabit (though XGB is the only significant consumer for now) should go as it is, and when we upgrade Rabit in XGB, the integration test there should be a guard to prevent regressions leading to the failure of XGB (not Rabit)

There are several cases we should cover 

1. XGB has breaking changes blocking the upgrading to the latest version of Rabit, integration test should not be able to run, so that we can fix XGB

2. Rabit has some bug not triggered by XGB, XGB just upgrades as long as we are sure integration test covers all involved functionalities 

3. Rabit has a bug triggered by XGB, integration test should detect that and we should fix in Rabit

I think it is the common practice in other projects, 

1. Hadoop 3 is very different with Hadoop 2, so Spark has to bring a lot of changes to upgrade to that

2. Parquet/Orc has several bugs all the way but not triggered by Spark, Spark just upgrade as usual

3. Orc has a bug, which is discovered by some integration test in some company, makes Spark cannot upgrade to the latest version, then Spark postpone the upgrading and Orc folks in Spark community went back to fix and added a test to guard